rllib/policy/eager_tf_policy.py

```diff
@@ -329,15 +332,29 @@ def compute_actions(self,
                     self, self.model, input_dict, self.observation_space,
                     self.action_space, self.config)
 
-            fetches = {}
+            # Override `action` with exploration action.
+            if explore and self.exploration:
+                action = self.exploration.get_exploration_action(
+                    action,
+                    self.model,
+                    action_dist=self.dist_class,
+                    explore=True,
+                    time_step=time_step or self.global_timestep)
+                self.last_exploration_info = self.exploration.get_info()
```

Comment:
Should we also set logp to None here?

Reply:
👍

Reply:
For DQN/Epsilon, it doesn't matter as the sampler returns None for it anyway. Interesting for PG methods where one could add exploration (e.g. ActionNoise) on top of the dist-sampling. Then again, for the training step (where action log is needed) probably irrelevant. I set it to None, nevertheless.